app/presenters/menu/default_menu.rb

```diff
@@ -266,35 +266,45 @@ def settings_menu_section
         ], :top_right)
       end
 
-      def help_menu_section
-        menu = {
-          :documentation => {
-            :title => N_('Documentation'),
-            :href  => '/support/index?support_tab=about'
-          },
-          :product       => {
-            :title => ::Settings.docs.product_support_website_text,
-            :href  => ::Settings.docs.product_support_website
-          },
-          :about         => {
-            :title => N_('About')
-          }
-        }.map do |key, value|
-          Menu::Item.new(key,
-                         help_menu_field(key, :title, value[:title]),
-                         key.to_s,
-                         {:feature => key.to_s},
-                         help_menu_field(key, :href, value[:href]),
-                         help_menu_field(key, :type, value[:type]),
-                         nil,
-                         value)
-        end
+      def help_documentation
+        help_menu_item(:documentation, :title => N_('Documentation'),
+                                       :href  => '/support/index?support_tab=about')
+      end
 
-        Menu::Section.new(:help, N_('Help'), 'pficon pficon-help', menu, :help)
+      def help_product
+        help_menu_item(:product, :title => ::Settings.docs.product_support_website_text,
+                                 :href  => ::Settings.docs.product_support_website)
+      end
+
+      def help_about
+        help_menu_item(:about, :title => N_('About'))
+      end
+
+      def help_menu_items
+        [
+          help_documentation,
+          help_product,
+          help_about,
+        ]
+      end
```

Comment:
This feels like a little bit of overkill to me, do we really have to use methods as variables?

Reply:
Yes, the new menu is using the items independently, not in order.

All this was to allow for inclusion of individual items separately.

Reply:
( https://github.com/ManageIQ/manageiq-ui-classic/pull/6963/commits/aac1c33f1517cac120f327074088a7d755bd2ef0 )